### PR TITLE
Create separate plugin for barometer

### PR DESCRIPTION
**Pull request by EliaTarasov**
Move barometer calculation to separate plugin.

### CODE REVIEW

**Comment by EliaTarasov** [comment]
Logs:
1. [master](https://review.px4.io/plot_app?log=8e9c7c75-11ae-40bd-b89f-867a28d7e625)
2. [pr](https://review.px4.io/plot_app?log=32b20ec5-257e-4e6a-9ae1-f493dcd7bef6)

**Comment by EliaTarasov** [comment]
@TSC21 Could you please review?

**Review by TSC21 — CHANGES_REQUESTED** [review]
All SDF models need to include this plugin.

File: CMakeLists.txt [hunk 1]
```diff
@@ -262,6 +262,7 @@ set(sensor_msgs
   msgs/Range.proto
   msgs/SITLGps.proto
   msgs/OpticalFlow.proto
+  msgs/Barometer.proto
```
Comment (TSC21) [inline CMakeLists.txt]: Barometer is the sensor. The data should be named depending on the type of data being propagated. In this case, something like `Pressure` (similar to ROS `FluidPressure`) makes more sense.

Comment (EliaTarasov) [reply CMakeLists.txt]: Agree, it's done.

**Comment by EliaTarasov** [comment]
@TSC21 done.

**Review by TSC21 — APPROVED** [review]
LGTM

**Comment by EliaTarasov** [comment]
@TSC21 I rebased it.

**Comment by TSC21** [comment]
@EliaTarasov just a remark: you are the author of the plugins but some of the code you are bringing to the plugins was already created by others in the mavlink interface. Is always good to give the credits to the initial authors ;)

**Pull request merged**